### ISSUE TITLE
Feature Request: Props to disabled tap to slide

### ISSUE THREAD

**Issue by johnlim5847** [Sharcoux/slider#6]
Hi, is it possible to have an option to disable tap the slider to change value? meaning the user has to drag the slider to change the value and not tap anywhere in the slider track to change the value.

**Comment by Sharcoux** [comment]
That's a good idea to make it a props. I hesitated a long time about the behavior it should have towards it. I'll make an update. Thanks for the idea.

**Comment by Sharcoux** [comment]
Can you check the new version 5.0.0 ?
Tell me if it suits your need and don't hesitate to open an issue if you encounter any issue with that new version.

**Comment by johnlim5847** [comment]
Hey, @Sharcoux thanks for getting back so quickly. I noticed there's a debug console.log at https://github.com/Sharcoux/slider/blob/master/src/hooks/useDrag.ts#L21

and is it intended not to call `onSlidingStart ` and `onSlidingComplete` when `slideToTap` is `false`? because in my use case I increased the size of `thumbSize` when the user is sliding and back to default size when the user finished sliding.

Also, is it possible to call `onSlidingComplete` when the user releases outside of the slider track?
